### DISCUSSION TITLE
Add middle mouse scrolling ability

### DISCUSSION THREAD

**Discussion by Shad0wSeven** [minbrowser/min#1503] [Ideas]
Adding this sort of integration, where after pressing middle mouse, allowing users to scroll automatically like its done in FF and Chromium, would be very useful.

![image](https://user-images.githubusercontent.com/19739712/107844242-9bb04d80-6d86-11eb-9c29-3b74ba4fd514.png)

**Comment by yurikhan** [comment]
I am interested in this. Chromium implements this on Windows only; would be nice to have it on Linux, too.
